specification/ml/explain_data_frame_analytics/MlExplainDataFrameAnalyticsRequest.ts

```diff
@@ -65,7 +65,7 @@ export interface Request extends RequestBase {
      * perform one of the following types of analysis: classification, outlier
      * detection, or regression.
      */
-    analysis: DataframeAnalysisContainer
+    analysis?: DataframeAnalysisContainer
```

Comment:
Documented as required, however there is a REST API test that makes a successful request without this field.